WalletWasabi.Gui/Controls/WalletExplorer/CoinInfoTabView.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui" 
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
+  xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui" 
+  xmlns:walexp="clr-namespace:WalletWasabi.Gui.Controls.WalletExplorer;assembly=WalletWasabi.Gui" 
+  xmlns:converters="clr-namespace:WalletWasabi.Gui.Converters;assembly=WalletWasabi.Gui" 
+  xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity" 
+  xmlns:behaviors="clr-namespace:WalletWasabi.Gui.Behaviors;assembly=WalletWasabi.Gui" x:Class="WalletWasabi.Gui.Controls.WalletExplorer.CoinInfoTabView">
+  <UserControl.Styles>
+    <Style Selector="TextBlock">
+      <Setter Property="VerticalAlignment" Value="Center" />
+      <Setter Property="FontSize" Value="{DynamicResource FontSizeLarge}" />
+    </Style>
+    <Style Selector="TextBox">
+      <Setter Property="FontSize" Value="{DynamicResource FontSizeLarge}" />
+    </Style>
+    <Style Selector="TextBox.Transparent">
+      <Setter Property="Background" Value="Transparent" />
+    </Style>
+  </UserControl.Styles>
+  <controls:GroupBox Title="{Binding Title}" DataContext="{Binding Coin}" Padding="20" Margin="10">
+    <Grid HorizontalAlignment="Left" ColumnDefinitions="150, *" RowDefinitions="Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto">
```

Comment:
On Wallet Info tab there is a different design. I would use the same with the following modifications:

- Instead of the wallet name, display the coin name as displayed on tab header.
- Instead of Keys => Coin details. 

![image](https://user-images.githubusercontent.com/9844978/74009008-9dc91a80-4982-11ea-9a1c-0ae8c8ac94db.png)